### PR TITLE
Prettier Upgrade

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/src/business/entities/Attorney.js [hunk 1]
```diff
@@ -30,33 +30,17 @@ joiValidationDecorator(
...
-    admissionsDate: joi
-      .date()
-      .iso()
-      .max('now')
-      .required(),
-    alternateEmail: joi
-      .string()
-      .optional()
-      .allow(null),
+    additionalPhone: joi.string().optional().allow(null),
```
Comment (matthopson) [inline shared/src/business/entities/Attorney.js:33]: Something weird is going on between the linter and the linter...
```
33:25  error  Replace `.string().optional()` with `⏎······.string()⏎······.optional()⏎······`                 prettier/prettier
  34:24  error  Replace `.date().iso().max('now')` with `⏎······.date()⏎······.iso()⏎······.max('now')⏎······`  prettier/prettier
  35:24  error  Replace `.string().optional()` with `⏎······.string()⏎······.optional()⏎······`                 prettier/prettier
  41:18  error  Replace `.string().optional()` with `⏎······.string()⏎······.optional()⏎······`                 prettier/prettier
  43:26  error  Replace `.string().optional()` with `⏎······.string()⏎······.optional()⏎······`                 prettier/prettier
```

Comment (matthopson) [reply shared/src/business/entities/Attorney.js:33]: Oh, is it because prettier isn't being updated in package.json in this PR?

Comment (ericsorenson) [reply shared/src/business/entities/Attorney.js:33]: I accidentally reverted in one commit.

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/entities/Attorney.js (hunk 1 above)
Comment (matthopson) [reply shared/src/business/entities/Attorney.js:33]: That would do it! 😅

**Review by matthopson — APPROVED** [review]

**Pull request merged**